modules/tinymce/CHANGELOG.md

```diff
@@ -6,47 +6,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `allow` as a valid attribute for the `iframe` element in the editor schema. #TINY-8939
 - New `search` field in the `MenuButton` that shows a search field at the top of the menu, and refetches items when the search field updates. #TINY-8952
 
 ### Improved
 - The formatter can now apply a format to a `contenteditable="false"` element by wrapping it. Configurable using the `format_noneditable_selector` option. #TINY-8905
-- The autocompleter now supports a multiple character trigger using the new `trigger` configuration #TINY-8887
-- The formatter will now apply some inline formats like color and font size to list item elements when the entire item content is selected. #TINY-8961
-- The plugin lists in the Help dialog are now sorted alphabetically. #TINY-9019
+- The autocompleter now supports a multiple character trigger using the new `trigger` configuration. #TINY-8887
+- The formatter now applies some inline formats, such as color and font size, to list item elements when the entire item content is selected. #TINY-8961
+- The available plugin lists in the Help dialog are now sorted alphabetically. #TINY-9019
 - Alignment can now be applied to more types of embedded media elements. #TINY-8687
 
 ### Changed
-- The `@menubar-row-separator-color` oxide variable no longer affects the divider between the Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubars. #TINY-8632
-- The `@toolbar-separator-color` oxide variable now exclusively affects the color of the separator between the Menubar and Toolbar. #TINY-8632
-- The available premium plugins listed in the Help dialog are no longer translated. #TINY-9019
-
-### Fixed
-- The Autolink plugin did not work when the text nodes in the content were fragmented #TINY-3723
-- Fixed various incorrect types on public APIs found while enabling TypeScript strict mode #TINY-8806
-- Text returned from `editor.getContent({format: 'text'})` would differ with blank lines between some browsers #TINY-8579
-- The editor focused via the `auto_focus` option was not scrolled into the viewport #TINY-8785
-- Adding spaces just after a `contenteditable="false"` block would not work properly in some situations #TINY-8814
-- Elements with only custom attributes starting with `data-` would sometimes be removed when they shouldn't #TINY-8755
-- Selecting a figure with `class="image"` would incorrectly highlight the link toolbar button #TINY-8832
+- The `@menubar-row-separator-color` oxide variable no longer affects the divider between the Menubar and Toolbar. It only controls the color of the separator lines drawn in multiline Menubars. #TINY-8632
+- The `@toolbar-separator-color` oxide variable now affects the color of the separator between the Menubar and Toolbar only. #TINY-8632
+- Available Premium plugins are listed by name in the Help dialog. These names are no longer translated. #TINY-9019
```

Comment:
This doesn't make sense... why do we have a statement here as the first sentence? The previous entry was more correct/better here by the looks.

Reply:
The original sentence has three components.

1. a 9-word noun phrase: *The available premium plugins listed in the Help dialog*

2. The English copula in the third-person plural simple present: *are*

3. An adverbial phrase: *no longer translated*

That 9-word noun phrase is not easy to parse.

*The available premium plugins* is simple enough. And easy to keep in the head.

*listed in the Help dialog* is a multi-word conditional that modifies the state of this simple object. To use the technical term, this is a **subject complement**.

It calls for the copular verb between the simple noun phrase and the modifying phrase. In this case it would be the _that are_ inflection. And this would make the distinction between the object and its complement clear.

Except we can’t do that. Because the entire noun phrase (the noun phrase and its subject complement) is, itself, in a particular state. And that state — *no longer translated* — can only be applied to the object by the copular verb (in this case the *are* form).

Having, in effect, two subject complements in a row makes for a difficult-to-parse sentence.

My re-write is an attempt to obviate this.

A first sentence that documents the thing being discussed.

1. a three word noun phrase: *Available Premium plugins*

2. The English copula in the third-person plural simple present: *are*

3. The subject complement that modifies the noun: *listed by name in the Help dialog.*

A second sentence that uses a referent to the first sentence.

1. A referent and a noun: *These names*

2. The English copula in the third-person plural simple present: *are*

3. The subject complement that modifies the noun: *no longer translated.*

By setting up what ‘names’ are in the first sentence, I can reference them in the second sentence while, simultaneously, clearly separating the two subject complements.

I set up a complex state: *Available Premium plugins listed by name in the Help dialog.*

And then collapse that entire complex state into one referent word — *names*. As a consequence, it is much easier to see what the second sentence’s subject complement (the thing that changes the state of this single word) applies to.

Put another way, the re-write is an attempt at verbal show-and-tell.

1. See this complicated pile here? ::waves at said complicated pile::

2. We did *this* ::points to ‘this’::

3. to **it**. ::points back at pile::

(note the linguistic simplifying of ‘the complicated pile’ down to a pronoun, ‘it’)

Reply:
While I'm not going to argue that... as a developer (who is the target audience here) this new wording is only confusing to me.

Reply:
It's readable to me.

Reply:
Following discussions elsewhere, a further re-write has been made.